automod/flagstore/flagstore_mem.go

```diff
@@ -23,13 +23,8 @@ func (s MemFlagStore) Get(ctx context.Context, key string) ([]string, error) {
 }
 
 func (s MemFlagStore) Add(ctx context.Context, key string, flags []string) error {
-	v, ok := s.Data[key]
-	if !ok {
-		v = []string{}
-	}
-	for _, f := range flags {
-		v = append(v, f)
-	}
+	v, _ := s.Data[key]
```

Comment:
If `key` is not in `s.Data`, then `v` will be `[]string(nil)`, which is suitable for using directly in the `append` that follows. All the other stuff is extraneous.